api/src/main/resources/application.yml

```diff
@@ -54,6 +54,15 @@ message-processor:
   minimum-lock-time: "PT1M"
   maximum-lock-time: "PT60M"
 
+new-payment-cycle-schedule:
+  # run at half-past the hour from 02:30 TO 05:30
```

Comment:
Why only at this time? I thought when we notify the user we can specify sociable times them.

Reply:
These times were plucked out of thin air, tbh. I figured we need to run multiple times per day but once we've processed everything for that day there won't be any more, so there's no point running throughout the day. I wanted to ensure we avoid any issues of the clocks changing, hence starting from 2:30, but allow most of the day for processing.